capstone/capapi/case_views.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+from django.http import HttpResponse
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import renderers, viewsets, mixins, filters as rs_filters
+
+from . import permissions, resources, serializers, filters
+from capdb import models
+
+logger = logging.getLogger(__name__)
+
+
+class JSONResponse(HttpResponse):
```

Comment:
Is this redundant of https://docs.djangoproject.com/en/1.11/ref/request-response/#jsonresponse-objects ?

If it's doing something different, maybe add a comment explaining.